packages/backend-for-frontend/src/services/selfcareService.ts

```diff
@@ -90,8 +92,8 @@ export function selfcareServiceBuilder(
     }: WithLogger<BffAppContext>): Promise<bffApi.SelfcareInstitution[]> {
       logger.info(`Retrieving Institutions for User ${userId}`);
 
-      const institutions = await selfcareV2Client.getInstitutionsUsingGET({
-        queries: { userIdForAuth: userId },
+      const institutions = await selfcareV2UsersClient.v2getUserInstitution({
+        queries: { userId },
```

Comment:
perhaps the institutionId should be added here together with the userId?

`{userId, institutionId: selfacreId}`